### PR TITLE
Fix capacities as well

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
**What type of PR is this?**
 
/kind failing-test
 /kind flake

**What this PR does / why we need it**:
Fixing the capacity to ensure scheduling works fine.
/sig scheduling

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
Hopefully the last change for fixing  #69989, I am updating the node capacity as well to fix the failing e2e. I am still going to monitor #69989
/cc @Huang-Wei 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/scheduling/priorities.go [hunk 1]
```diff
@@ -466,6 +466,8 @@ func updateNodeAllocatable(c clientset.Interface, nodeName string, memory, cpu i
 	if err != nil {
 		return err
 	}
+	node.Status.Capacity[v1.ResourceMemory] = *resource.NewQuantity(memory, resource.BinarySI)
```
Comment (Huang-Wei) [inline test/e2e/scheduling/priorities.go]: Two concerns:

1. The function is named as `updateNodeAllocatable`, but the logic inside is updating `Capacity` as well. It should be consistent.
1. It seems the **capacity** field is not set back to original value after testing?

Comment (ravisantoshgudimetla) [reply test/e2e/scheduling/priorities.go]: Yup, both are right, I am going to address them.

**Review by Huang-Wei — APPROVED** [review]
/lgtm

**Pull request merged**